src/client/Microsoft.Identity.Client/Extensibility/MsaPassthroughConfig.cs

```diff
@@ -17,14 +17,13 @@ public static class MsaPassthroughConfig
 
         /// <summary>
         /// Declares that the app is MSA-Passthrough enabled (Microsoft internal only). This is a legacy 
-        /// feature. Required for the functionality of some brokers.
+        /// feature. Required for the functionality of some brokers, like WAM.
```

Comment:
@jmprieur - for the naming and comments. Please note the namespace used.
Could maybe rename it to `WithLegacyMsaPassthrough` ...